<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWinMouseConfigForm</class>
 <widget class="QWidget" name="KWinMouseConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>416</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,5">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Double-click:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>coTiDbl</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="KComboBox" name="coTiDbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Behavior on &lt;em&gt;double&lt;/em&gt; click into the titlebar.</string>
       </property>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">Maximize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">Maximize (vertical only)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">Maximize (horizontal only)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">Minimize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">Shade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">On All Desktops</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="@item:inlistbox behavior on double click">Nothing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Wheel event:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>coTiAct4</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="KComboBox" name="coTiAct4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Handle mouse wheel events</string>
       </property>
       <item>
        <property name="text">
         <string>Raise/Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shade/Unshade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Maximize/Restore</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Keep Above/Below</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Move to Previous/Next Desktop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Change Opacity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Switch to Window Tab to the Left/Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>In this column you can customize mouse clicks into the titlebar or the frame of an active window.</string>
       </property>
       <property name="text">
        <string>Active</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>In this column you can customize mouse clicks into the titlebar or the frame of an inactive window.</string>
       </property>
       <property name="text">
        <string>Inactive</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="whatsThis">
        <string>In this row you can customize left click behavior when clicking into the titlebar or the frame.</string>
       </property>
       <property name="text">
        <string>Left button:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="KComboBox" name="coTiAct1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Behavior on &lt;em&gt;left&lt;/em&gt; click into the titlebar or frame of an &lt;em&gt;active&lt;/em&gt; window.</string>
       </property>
       <item>
        <property name="text">
         <string>Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toggle Raise &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minimize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operations Menu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Start Window Tab Drag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="KComboBox" name="coTiInAct1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Behavior on &lt;em&gt;left&lt;/em&gt; click into the titlebar or frame of an &lt;em&gt;inactive&lt;/em&gt; window.</string>
       </property>
       <item>
        <property name="text">
         <string>Activate &amp; Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Activate &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Activate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toggle Raise &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minimize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operations Menu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Start Window Tab Drag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="whatsThis">
        <string>In this row you can customize middle click behavior when clicking into the titlebar or the frame.</string>
       </property>
       <property name="text">
        <string>Middle button:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="KComboBox" name="coTiAct2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Behavior on &lt;em&gt;middle&lt;/em&gt; click into the titlebar or frame of an &lt;em&gt;active&lt;/em&gt; window.</string>
       </property>
       <item>
        <property name="text">
         <string>Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toggle Raise &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minimize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operations Menu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Start Window Tab Drag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="KComboBox" name="coTiInAct2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Behavior on &lt;em&gt;left&lt;/em&gt; click into the titlebar or frame of an &lt;em&gt;inactive&lt;/em&gt; window.</string>
       </property>
       <item>
        <property name="text">
         <string>Activate &amp; Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Activate &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Activate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toggle Raise &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minimize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operations Menu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Start Window Tab Drag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="whatsThis">
        <string>In this row you can customize right click behavior when clicking into the titlebar or the frame.</string>
       </property>
       <property name="text">
        <string>Right button:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="KComboBox" name="coTiAct3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Behavior on &lt;em&gt;right&lt;/em&gt; click into the titlebar or frame of an &lt;em&gt;active&lt;/em&gt; window.</string>
       </property>
       <item>
        <property name="text">
         <string>Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toggle Raise &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minimize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operations Menu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Start Window Tab Drag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="KComboBox" name="coTiInAct3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Behavior on &lt;em&gt;left&lt;/em&gt; click into the titlebar or frame of an &lt;em&gt;inactive&lt;/em&gt; window.</string>
       </property>
       <item>
        <property name="text">
         <string>Activate &amp; Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Activate &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Activate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Raise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toggle Raise &amp; Lower</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minimize</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shade</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Operations Menu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Start Window Tab Drag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="whatsThis">
          <string>Behavior on &lt;em&gt;left&lt;/em&gt; click onto the maximize button.</string>
         </property>
         <property name="text">
          <string>Left button:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>leftClickMaximizeButton</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_9">
         <property name="whatsThis">
          <string>Behavior on &lt;em&gt;middle&lt;/em&gt; click onto the maximize button.</string>
         </property>
         <property name="text">
          <string>Middle button:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>middleClickMaximizeButton</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_8">
         <property name="whatsThis">
          <string>Behavior on &lt;em&gt;right&lt;/em&gt; click onto the maximize button.</string>
         </property>
         <property name="text">
          <string>Right button:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>rightClickMaximizeButton</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="KComboBox" name="leftClickMaximizeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>Behavior on &lt;em&gt;left&lt;/em&gt; click onto the maximize button.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KComboBox" name="middleClickMaximizeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>Behavior on &lt;em&gt;middle&lt;/em&gt; click onto the maximize button.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="KComboBox" name="rightClickMaximizeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="whatsThis">
          <string>Behavior on &lt;em&gt;right&lt;/em&gt; click onto the maximize button.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0" colspan="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Titlebar &amp; Frame</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Titlebar</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Maximize Button</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>coTiDbl</tabstop>
  <tabstop>coTiAct4</tabstop>
  <tabstop>coTiAct1</tabstop>
  <tabstop>coTiInAct1</tabstop>
  <tabstop>coTiAct2</tabstop>
  <tabstop>coTiInAct2</tabstop>
  <tabstop>coTiAct3</tabstop>
  <tabstop>coTiInAct3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
